packages/fuels/package.json

```diff
@@ -14,24 +14,24 @@
   },
   "exports": {
     ".": {
+      "types": "./dist/index.d.ts",
```

Comment:
@petertonysmith94 Was it you who changed the order of these properties at some point?